en-EN.js

```diff
@@ -432,151 +432,151 @@ module.exports = {
                 },
                 graph: {
                     meta: {
-                        title: "Graphique",
-                        description: "Affiche un graphique."
+                        title: "Graphic",
+                        description: "Displays a graph."
                     },
                     elements: {
                         messages: {
-                            title: "Messages envoyés",
-                            description: "Graphique sur vos messages envoyés"
+                            title: "Sent messages",
+                            description: "Graphic on your sent messages"
                         },
                         voice: {
-                            title: "Temps en vocal",
-                            description: "Graphique sur votre activité vocale"
+                            title: "Vocal time",
+                            description: "Graphic on your voice activity"
                         },
                         reactionsGiven: {
-                            title: "Réactions données",
-                            description: "Graphique sur vos réactions données"
+                            title: "Added reactions",
+                            description: "Graphic on your added reactions"
                         },
                         deletedMessages: {
-                            title: "Messages supprimés",
-                            description: "Graphique sur vos messages supprimés"
+                            title: "Deleted messages",
+                            description: "Graphic on your deleted messages"
                         }
                     }
                 },
                 percentage: {
                     meta: {
-                        title: "Pourcentage",
-                        description: "Affiche un pourcentage + barre de progression."
+                        title: "Percentage",
+                        description: "Displays a percentage + progress bar."
                     },
                     elements: {
                         involvement: {
                             title: "Engagement",
-                            text: "Taux de présence sur le serveur/au cours du mois dernier.",
-                            description: "Présence en % sur ce serveur"
+                            text: "Presence rate on the server /in the last month.",
+                            description: "Presence in % on this server"
                         },
                         firstPlaceMessages: {
                             title: "Progression #1",
-                            text: "Progression pour devenir #1 du/serveur en terme de messages.",
-                            description: "Progression pour devenir #1 du serveur"
+                            text: "Progression to become #1 of the/server in terms of messages.",
```

Comment:
"progression" -> "progress" fait plus naturel
"to become" -> "to reach" me parrait mieux
"of the/server in terms of messages." -> "in the/message leaderboard." aussi
*(Et même chose sur le component plus bas)*